metrics.go

```diff
@@ -22,6 +22,8 @@ const (
 	// Supported metric types.
 	MetricTypeGauge    MetricType = "gauge"
 	MetricTypeAbsolute MetricType = "absolute"
+	PlanRequired       string     = "required"
```

Comment:
Should these have their own type?

Anyway, they shouldn't be in the "supported metric types" section unless they are actually metric types.